src/nickel.rs

```diff
@@ -178,6 +181,20 @@ impl<D: Sync + Send + 'static> Nickel<D> {
     pub fn keep_alive_timeout(&mut self, timeout: Option<Duration>){
         self.keep_alive_timeout = timeout;
     }
+
+    /// Set the number of threads the server should use
+    ///
+    /// # Performance
+    ///
+    /// Increasing this can help with thread exhaustion. Setting this
+    /// to `None` will use the default value.
+    ///
+    /// # Default
+    ///
+    /// The default value is 5 threads
```

Comment:
I didn't realize it was a calculation, was just going by the logs. I'll update the docs.